src/actions/loadSpace.js

```diff
@@ -42,13 +43,19 @@ export const loadSpace = payload => dispatch => {
   window.ipcRenderer.once(LOADED_SPACE_CHANNEL, (event, response) => {
     switch (response) {
       case ERROR_GENERAL:
-        toastr.error(ERROR_MESSAGE_HEADER, ERROR_LOADING_MESSAGE);
+        toastr.error(
```

Comment:
There's a lot of repetition here. Do you think we could have a function to factor it out? Just a thought.